media/main.js

```diff
@@ -5,6 +5,22 @@ const prJson = {
 while (!$) {
   console.log("no $");
 }
+
+function parseMarkdown(markdownText) {
```

Comment:
Do we need an internal markdown parser?
There is this repo with 27k stars that only requires importing it's script
https://github.com/markedjs/markedhttps://github.com/markedjs/marked

Reply:
It's there simply because it works! 

The repo you're linking to is an interesting suggestion, but I tried doing this and it doesn't work. It's something on my side for sure, but I don't know how to handle it becuase it's JQuery lol. If you wanna do the fix quickly (if it's quick) go ahead! I'd just prefer us to do other high leverage stuff first. 

Attaching the image for documenting purposes. 

![Screen Shot 2022-03-12 at 4 57 52 PM](https://user-images.githubusercontent.com/8325094/158036493-7985ffdf-14dd-47a2-813a-02bce61fe044.png)